src/LinuxTracing/PerfEventOpen.h

```diff
@@ -103,6 +103,9 @@ static constexpr uint64_t SAMPLE_REGS_USER_SP_IP_ARGUMENTS =
 //  some setting.
 static constexpr uint16_t SAMPLE_STACK_USER_SIZE = 65000;
 
+// Arbitrary small value, that is still large enough to contain the complete last frame.
+static constexpr uint16_t SAMPLE_STACK_USER_SIZE_ON_CALLCHAINS = 128;
```

Comment:
As this is used in a `struct` called `perf_event_sample_stack_user_128bytes`, and as the constant below is called `SAMPLE_STACK_USER_SIZE_8BYTES`, I would suggest `SAMPLE_STACK_USER_SIZE_128BYTES`. I know it's uglier but I think it's more consistent with the rest of the code.